tools/serverpod_cli/lib/src/generator/dart/library_generators/class_generators/repository_classes.dart

```diff
@@ -1285,7 +1302,7 @@ class BuildRepositoryClass {
 
       methodBuilder
         ..docs.add(
-            '/// Remove the `${field.name}` field from the [$classFieldName]')
+            '/// Removes the `${field.name}` field from the [$classFieldName]')
```

Comment:
Comment: Perhaps we should add something about the relation here too.

Reply:
@Isakdl What do you have in mind?

A longer explanation (for which I might not be firm enough on the details), or just add "relation" to these 6 methods like 

```
'/// Removes the "`${field.name}`" relation from the [$classFieldName]'
```

(thus replacing the technical (or storage-oriented) "field" with the logical "relation".

Reply:
Updated the comments with the latest version we agreed upon in Slack.